lib/controllers/login.js

```diff
@@ -10,6 +10,7 @@ var forms = require('../forms');
 var helpers = require('../helpers');
 var oauth = require('../oauth');
 var loginWithOAuthProvider = require('../helpers/login-with-oauth-provider');
+var getLoginViewModel = require('../helpers/get-login-view-model');
```

Comment:
I'm not too happy about this *helper* file, and as you said yourself, this code should not even belong here. But in lack of a better structure at the moment, your suggestion is better and I fixed it (1923c19234ee0928e890f526f4be422e37fcaf73) :)